public_html/wp-content/mu-plugins/jetpack-tweaks/css-sanitization.php

```diff
@@ -222,3 +223,16 @@ function update_csstidy_safelist() {
 		$GLOBALS['csstidy']['all_properties'] = array_merge( $GLOBALS['csstidy']['all_properties'], $properties_for_csstidy );
 	}
 }
+
+/**
+ * Replace encoded &gt; in CSS with a > so that CSS rules are valid.
+ *
+ * Remove this once the root issue is fixed in https://github.com/Automattic/jetpack/issues/21603.
+ *
+ * @param string $css CSS pulled in from the Custom CSS post type.
+ *
+ * @return string
+ */
+function filter_custom_css( $css ) {
+	return str_replace( '&gt;', '>', $css );
```

Comment:
Are there any other characters this could happen to? Would it make sense to run something like `html_entity_decode()` here instead of a simple string replace?

Reply:
I'm going to merge this as-is, if other selectors become a problem we can make the switch then.